### ISSUE TITLE
Release notes/changelog

### ISSUE THREAD

**Issue by testn** [jnr/jffi#31]
Is it possible to provide release notes or changelog so that we know what have been changed in each version?

**Comment by headius** [comment]
It should be possible to view the version in github and see all commits. Is there value in us just pulling commit lines out for release notes?

**Comment by headius** [comment]
Fix or punt by 1.2.14.

**Comment by nirvdrum** [comment]
Answering the question posed on May 16, I've yet to find an easy way to see the list of commits between two tags. You can do it with some URL fiddling, but GitHub doesn't make it terribly easy to do. And then the commits appear in reverse chronological order, which gets confusing if commits build on top of each other.

**Comment by headius** [comment]
Yeah, I've been trying to find something that could easily generate github release notes for us, but they're all pretty crappy.
